config/samples/migration_v1alpha1_directpvmigrationprogress.yaml

```diff
@@ -3,7 +3,7 @@ kind: DirectVolumeMigrationProgress
 metadata:
   labels:
     controller-tools.k8s.io: "1.0"
-  name: directpvmigrationprogress-sample
+  name: directvolumemigrationprogress-sample
 spec:
   # Add fields here
   foo: bar
```

Comment:
Do we want actual spec fields here?

Reply:
Users won't create these, though will they? If these are only ever controller-generated, do we need samples?

Reply:
I was told to keep these up to date for users so unsure